test/integration/trigger.test.js

```diff
@@ -0,0 +1,202 @@
+'use strict';
+
+var chai = require('chai')
+  , Sequelize = require('../../index')
+  , expect = chai.expect
+  , Support = require(__dirname + '/../support')
+  , dialect = Support.getTestDialect()
+  , current = Support.sequelize;
+
+
+if(current.dialect.supports.tmpTableTrigger)
+{
+  describe(Support.getTestDialectTeaser('Model'), function() {
+    describe('trigger', function() {
+
+
+        it('should return output rows after insert', function() {
+          var self = this;
+
+          var triggerQuery = 'create trigger User_ChangeTracking on [users] for insert,update, delete \n' +
```

Comment:
Place this inside the innermost describe, so you don't have to duplicate it each time